### DISCUSSION TITLE
JSON schema draft-07 schema specification details

### DISCUSSION THREAD

**Discussion by kspmmitt** [json-schema-org/community#486] [Q&A]
Hello Community,

I am following draft-07 json schema specification to create schemas.

Is there any detailed description of specification for draft-07 that can be followed for creating schemas?

Rgds,
Kamal

**Comment by jdesrosiers** [comment]
You can find links to every released specification on the website. You'll find draft-07 at, https://json-schema.org/specification-links.html#draft-7.

However, that reference is more for people writing implementations that evaluated schemas than for people who just want to write schemas. You may find [Understanding JSON Schema](https://json-schema.org/understanding-json-schema/) more helpful. This reference covers all versions of JSON Schema. There are notes throughout to inform you of any difference in behavior from one version to another. Anything that doesn't say what version it applies to applies to all versions.

**Reply by kspmmitt** [reply]
Thanks.
